docs/developer-docs/latest/development/plugins/users-permissions.md

```diff
@@ -564,6 +564,66 @@ The use of `ngrok` is not needed.
   - **The redirect URL to your front-end app**: `http://localhost:3000/connect/linkedin/redirect`
 
 :::
+::: tab CAS
+
+#### Using ngrok
+
+A remote CAS server can be configured to accept `localhost` urls or you can run your own CAS server locally that accepts them.<br>
```

Comment:
```suggestion
A remote CAS server can be configured to accept `localhost` URLs or you can run your own CAS server locally that accepts them.

```

Adding a double-line break is better, as we try to inject as less HTML as possible in our Markdown files :-)